os/kernel/binary_manager/binary_manager_data.c

```diff
@@ -116,6 +116,10 @@ int binary_manager_register_ubin(char *name)
 	for (bin_idx = 1; bin_idx <= g_bin_count; bin_idx++) {
 		/* Already Registered */
 		if (!strncmp(BIN_NAME(bin_idx), name, strlen(name) + 1)) {
+			if (priority > BIN_PRIORITY(bin_idx)) {
+				/* Update higher priority for early loading */
```

Comment:
This is not always used for early loading.
If priority is higher than binary manager, it will be. But even if the priority is higher than before, but lower than binary manager, it will not be.
Isn't it?